addons/product/models/product_template.py

```diff
@@ -139,6 +139,11 @@ def _read_group_categ_id(self, categories, domain, order):
     can_image_1024_be_zoomed = fields.Boolean("Can Image 1024 be zoomed", compute='_compute_can_image_1024_be_zoomed', store=True)
     has_configurable_attributes = fields.Boolean("Is a configurable product", compute='_compute_has_configurable_attributes', store=True)
 
+    priority = fields.Selection([
+        ('0', 'Normal'),
+        ('1', 'Favorite'),
+    ], default='0', index=True, string="Priority")
```

Comment:
An index on a field with two values is usually a waste of space (go look for recommendations on indexing boolean columns).

If "favorites" are rare and you want to provide fast access to searches on that criteria, you should use a partial index.

Also is there any reason for this field not to be required?

Reply:
declared this way to be consistent with most priorities, especially in mrp, stock & purchase

Reply:
Don't forget to remove it